src/resources/compatibility.ts

```diff
@@ -1,6 +1,7 @@
 // Caution before editing - For latest builds, this module is replaced with emptiness and thus not imported (see build-scripts/bundle.js)
 import "core-js";
 import "regenerator-runtime/runtime";
+import "globalthis-polyfill";
```

Comment:
I don't think we need another package - `core-js` should already be polyfilling this?

https://github.com/zloirock/core-js#ecmascript-globalthis

Reply:
I'll test